demos/text_detection_demo/cpp_gapi/src/text_recognition.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <algorithm>
+#include <cmath>
+#include <numeric>
+#include <fstream>
+
+#include "text_recognition.hpp"
+
+bool fileExists(const std::string& filename) {
+    std::ifstream f(filename.c_str());
+    return f.good();
+}
+bool tryReadVocabFile(const std::string& filename, std::string& alphabet) {
```

Comment:
You can let `tryReadVocabFile()` decide how to interpret `FLAGS_m_tr_ss`. This function should just return alphabet string given `FLAGS_m_tr_ss`. In simplest case it should just return passed `FLAGS_m_tr_ss`.
Please add similar function to cpp version